Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/Fleet.hs

```diff
@@ -0,0 +1,31 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Domain.Types.Fleet where
+
+import Data.OpenApi (ToSchema)
+import EulerHS.Prelude hiding (id, state)
+
+data FleetOwnerLoginReq = FleetOwnerLoginReq
```

Comment:
These are request types, not domain types. So, they should be placed in `Domain.Action.Dashboard.Fleet.Registration` or even in dashboard  `CommonAPIs` lib`